pkg/expansion/system_test.go

```diff
@@ -83,10 +83,21 @@ func TestExpand(t *testing.T) {
 		},
 		{
 			name:      "expand with nil namespace returns error",
-			generator: fixtures.LoadFixture(fixtures.DeploymentNginx, t),
+			generator: fixtures.LoadFixture(fixtures.DeploymentNginxWithNs, t),
```

Comment:
don't we still want coverage to test `DeploymentNginx`?

Reply:
This is just a fixture to return a pod with a custom namespace. We also sitll use `DeploymentNginx` in other places.